### PR TITLE
Maven Dependency Version Updates

### PR DESCRIPTION
**Pull request by cboppert**
Addition to readme.

### CODE REVIEW

**Comment by cboppert** [comment]
Blocked until
https://github.com/lightstep/lightstep-tracer-java-common/pull/91
goes out

**Comment by cboppert** [comment]
Relies on (and blocked by) https://github.com/lightstep/lightstep-tracer-java-common/pull/91

File: lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import io.opentracing.Span;
 import io.opentracing.SpanContext;
 import io.opentracing.Tracer;
-import io.opentracing.propagation.TextMapExtractAdapter;
+import io.opentracing.propagation.TextMapAdapter;
```
Comment (frenchfrywpepper) [inline lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java]: This change seems unrelated to the PR

Comment (cboppert) [reply lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java]: It fixes a test which would break "mvn clean install"

Comment (frenchfrywpepper) [reply lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java]: I'm not sure I follow. I checked out master, updated OT to 0.32.0 and successfully ran mvn clean install. Maybe if you separate out the OT dependency upgrades from the formatting changes and the common version bumps and the version bumps of this jar it will be more clear?

Comment (cboppert) [reply lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java]: Okay I'll just undo it

Comment (cboppert) [reply lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java]: If updating OT to .32 works

**Comment by JulianGriggs** [comment]
@cboppert What is the status of this? It would be great to get this merged

**Comment by carlosalberto** [comment]
hey hey! Now that lightstep/lightstep-tracer-java-common#91 is done, it seems the remaining changes here are mostly style/spaces, no? (Other than the note for Example).

(Sorry @cboppert for the late review!)
